src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -352,6 +358,20 @@ def get_speed_bps(self, sub_id):
         # speed is reported in Mbps in the db
         return min(self.RFC1213_MAX_SPEED, speed * 1000000)
 
+    def get_if_type(self, sub_id):
+        """
+        :param sub_id: The 1-based sub-identifier query.
+        :return: integer representing a type according to textual convention
+
+        ethernetCsmacd(6), -- for all ethernet-like interfaces,
+                           -- regardless of speed, as per RFC3635
+        ieee8023adLag(161) -- IEEE 802.3ad Link Aggregate
+        """
+        oid = self.get_oid(sub_id)
+        if oid:
+            return IfTypes.ethernetCsmacd if oid < 1000 else IfTypes.ieee8023adLag
```

Comment:
>oid < 1000 [](start = 45, length = 10)

magic number. I suggest add new function in sonic-py-swsssdk/src/swsssdk/port_util.py